examples/bace/bace_dnn.py

```diff
@@ -1,101 +0,0 @@
-import sys
-import os
-import deepchem
-import tempfile, shutil
-import numpy as np
-import numpy.random
-from bace_datasets import load_bace
-from deepchem.utils.save import load_from_disk
-from deepchem.splits import SpecifiedSplitter
-from deepchem.data import Dataset
-from deepchem.hyper import HyperparamOpt
-from deepchem import metrics
-from deepchem.metrics import Metric
-from deepchem.utils.evaluate import Evaluator
-from deepchem.models.keras_models.fcnet import MultiTaskDNN
-from deepchem.models.keras_models import KerasModel
```

Comment:
these are gone -- I would have to go through the git time machine to find them if we really wanted them.

Reply:
No worries. It's not worth investing effort into deprecated models